includes/events/class-omise-event-charge-complete.php

```diff
@@ -44,68 +44,26 @@ class Omise_Event_Charge_Complete {
 	 * @return void
 	 */
 	public function handle( $data ) {
-		if ( 'charge' !== $data->object || ! isset( $data->metadata->order_id ) ) {
+		if ( 'charge' !== $data['object'] || ! isset( $data['metadata']['order_id'] ) ) {
```

Comment:
Same has to be done for other event classes i.e. `Omise_Event_Charge_Create` , ` Omise_Event_Charge_Capture`.

Reply:
@mayurkathale awesome catch! 👍 
Done